front/lib/resources/storage/models/files.ts

```diff
@@ -27,7 +27,6 @@ export class FileModel extends Model<
   declare fileName: string;
   declare fileSize: number;
   declare status: FileStatus;
-  declare sId: string;
```

Comment:
I would suggest continue storing the sId we just know it's a decodable one for this resource

In the future all resources will use these sId but we will continue storing them

Reply:
Why storing them if you can re-compute them?

Reply:
Seen IRL.